.github/workflows/build.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     with:
       curse-publish-task: ""
       maven-snapshots: true
+      java-version: 21
```

Comment:
shouldn't we be using java 17 for 1.20.1?

Reply:
`sourceCompatibility = targetCompatibility = compileJava.sourceCompatibility = compileJava.targetCompatibility = 17`